src/profile-logic/process-profile.js

```diff
@@ -985,7 +1014,8 @@ function _processCounters(
   // The gecko profile's process don't map to the final thread list. Use the stable
   // thread list to look up the thread index for the main thread in this profile.
   const mainThreadIndex = stableThreadList.findIndex(
-    (thread) => thread.name === 'GeckoMain' && thread.pid === mainThread.pid
+    (thread) =>
+      thread.name === 'GeckoMain' && thread.pid === `${mainThread.pid}`
```

Comment:
Nit: We could create a constant variable for the `${mainThread.pid}` string outside of the iteration so we don't create a lot of intermediate strings in this loop. 

Edit: Ah, also we are creating the same one when we are returning too.